testsuite/features/secondary/srv_restart_postgres.feature

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Restart the PostgreSQL service
+
+  Scenario: Restart the PostgreSQL database service
+    Given I am authorized for the "Admin" section
+    When I restart "postgresql.service" service on "server"
+    And I wait until "postgresql" service is active on "server"
+    And I am on the Systems overview page of this "sle_minion"
```

Comment:
After the Postgresql service is active, we need to be sure that Taskomatic has reconnected too.
Proposal: Look for a concrete message on the taskomatic logs.

Reply:
I see some errors in the taskomatic log file, but nothing that the connection was established again:

```bash
org.postgresql.util.PSQLException: This connection has been closed.
        at org.postgresql.jdbc.PgConnection.checkClosed(PgConnection.java:880) ~[postgresql.jar:42.2.25]
        at org.postgresql.jdbc.PgConnection.setAutoCommit(PgConnection.java:809) ~[postgresql.jar:42.2.25]
        at com.mchange.v2.c3p0.impl.NewProxyConnection.setAutoCommit(NewProxyConnection.java:1049) ~[c3p0-0.9.5.5.jar:0.9.5.5]
        at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.restoreOriginalAtributes(AttributeRestoringConnectionInvocationHandler.java:141) ~[quartz-2.
3.0.jar:?]
```